frontend/src/pages/landing-page/index.tsx

```diff
@@ -40,48 +39,114 @@ const LandingPage = (): JSX.Element => {
     rootMargin: "-70% 0px -30% 0px",
   });
 
-  const [activeNav, setActiveNav] = useState(1);
+  // HERO NUMBERS. DUMMY DATA TO BE REPLACED.
+  const [cellsHeroNum] = useState("100M+");
+  const [datasetsHeroNum] = useState("436");
+  const [donorsHeroNum] = useState("2.7k+");
```

Comment:
Probably out of context 😆  How do we expect these states to change? If not, constants might work?

Reply:
These will supposedly be connected to live data, however I'm not sure how that will be implemented exactly since these are strings. For example, perhaps a function will be written to convert 3,478 to "3.4k+", or 204,949,393 to "200M+"?

For now, I removed the "setHeroNum" functions since they weren't being used, and thus causing an error when trying to submit PR.